gammapy/analysis/tests/test_analysis.py

```diff
@@ -109,6 +112,25 @@ def test_analysis_1d():
     assert_allclose(dnde[-1].value, 5.382879e-21, rtol=1e-2)
 
 
+@requires_data()
+def test_exclusion_region(tmp_path):
+    config = AnalysisConfig.from_template("1d")
+    analysis = Analysis(config)
+
+    skydir = SkyCoord(83, 22, unit="deg", frame="icrs")
+    exclusion_region = CircleSkyRegion(center=SkyCoord(85, 23, unit="deg", frame="icrs"), radius=1 * u.deg)
+    exclusion_mask = Map.create(npix=(150, 150), binsz=0.05, skydir=skydir, proj="TAN", coordsys="CEL")
+    mask = exclusion_mask.geom.region_mask([exclusion_region], inside=False)
+    exclusion_mask.data = mask.astype(int)
+    filename = tmp_path / "exclusion.fits"
+    exclusion_mask.write(filename)
+    config.datasets.background.exclusion = str(filename)
```

Comment:
Do you need to put `str(filename)` here. It would be nice if assigning `Path` objects works transparently, converting to `str` on assignment. Does it already?

Reply:
Done